docs/tutorials/solving-maxcut-with-reduced-qubit-requirements-using-pauli-correlation-encoding.ipynb

```diff
@@ -74,7 +74,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        passed through the nonlinear function tanh(alpha * \prod_i). The loss function is
</pre><p>I think removing the backslash here would prevent the syntax warning. Then just run the notebook up to this cell –&nbsp;no need to run the entire thing again.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2980/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>